campaignion_donation_amount/campaignion_donation_amount.info

```diff
@@ -0,0 +1,10 @@
+name = Campaignion donation amount total token
+description = Define a webform submission token that calculates the sum of two donation amounts.
```

Comment:
I think the description doesn’t quite cover what the module does?

Reply:
True. Updated in the version that I just pushed.

Reply:
Better, thanks! For me "donation amount components" sound as if that was a component type though. Maybe it could be changed to something like "components used as donation amount"?

Reply:
They are “components” but not “component types”. I’d say `$component` is one specific configuration while `$component['type']` specifies which component plugin to use.

Reply:
I agree with the technical differentiation but that connotation does not get across in an English sentence :wink:

Reply:
How does the difference between “components” and “component types” not come across in english?